Gopkg.lock

```diff
@@ -2,38 +2,31 @@
 
 
 [[projects]]
-  digest = "1:218ae0d3f01ab221e2a62493a88e72189fb4941b22f60984277c22f664d34c46"
   name = "cloud.google.com/go"
   packages = ["compute/metadata"]
-  pruneopts = "NUT"
```

Comment:
update `dep` to v0.5 and some of these diffs will go away

Reply:
Thanks for the tip, done.